examples/tutorials/workgraphs/descriptors_filter_qe.ipynb

```diff
@@ -0,0 +1,474 @@
+    "        )\n",
+    "\n",
+    "        final_structures[f\"structs{i}\"] = descriptors_calc.outputs.xyz_output\n",
+    "\n",
+    "    split_task_inputs = {\n",
+    "        \"trajectory_data\": final_structures,\n",
+    "        \"config_types\": Str(\"\"),\n",
+    "        \"n_samples\": Int(len(final_structures.keys())),\n",
+    "        \"prefix\": Str(\"\"),\n",
+    "        \"scale\": Float(1.0e5),\n",
+    "        \"append_mode\": Bool(False),\n",
+    "    }\n",
+    "\n",
+    "    split_task = wg.add_task(create_aiida_files, inputs=split_task_inputs)\n",
+    "\n",
+    "    qe_inputs = {\n",
+    "        \"test_file\": split_task.outputs.test_file,\n",
+    "        \"train_file\": split_task.outputs.train_file,\n",
+    "        \"valid_file\": split_task.outputs.valid_file,\n",
+    "        \"calc_metadata\":  Dict({\n",
+    "                \"options\": {\n",
+    "                    \"resources\": {\n",
+    "                        \"num_machines\": 1,\n",
+    "                        'num_mpiprocs_per_machine': 32,\n",
+    "                    },\n",
+    "                    'max_wallclock_seconds': 3600,\n",
+    "                    'queue_name': 'scarf',                  \n",
+    "                    'qos': 'scarf',\n",
+    "                    'environment_variables': {},\n",
+    "                    'withmpi': True,                       \n",
+    "                    'prepend_text': '''\n",
+    "                    module purge\n",
+    "                    module use /work4/scd/scarf562/eb-common/modules/all\n",
+    "                    module load amd-modules\n",
+    "                    module load QuantumESPRESSO/7.2-foss-2023a\n",
+    "                    ''',\n",
+    "                    'append_text': ''  \n",
+    "                }\n",
+    "            }),\n",
+    "            \"kpoint_mesh\": List([1,1,1])\n",
+    "        }\n",
+    "\n",
+    "    qe_task = wg.add_task(qe, name=\"QE_workflow\", **qe_inputs)\n",
+    "\n",
+    "    qe_files = {\n",
+    "        \"test_file\": qe_task.outputs.test_file,\n",
+    "        \"train_file\": qe_task.outputs.train_file,\n",
+    "        \"valid_file\": qe_task.outputs.valid_file\n",
+    "    }\n",
+    "\n",
+    "    training_files = wg.add_task(create_train_file, **qe_files)\n",
+    "\n",
+    "    train_task = wg.add_task(\n",
+    "        trainCalc,\n",
+    "        mlip_config = training_files.outputs.JanusConfigfile,\n",
+    "        code = janus_code,\n",
+    "        metadata={'options': {\"resources\": {\"num_machines\": 1}}},\n",
+    "        fine_tune =True,\n",
+    "        foundation_model= model\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f3c72ca",
+   "metadata": {},
+   "source": [
+    "Visualise the workgraph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0d119cad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wg\n"
```

Comment:
This seems to show `create_aiida_files` as disconnected from the first half. Does the full provenance show them as connected?